help/destinations/catalog/email-marketing/oracle-eloqua-api.md

```diff
@@ -0,0 +1,154 @@
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Oracle Eloqua] Update a Contact REST API](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/op-api-rest-1.0-data-contact-id-put.html), which allows you to update identities within a segment into [!DNL Oracle Eloqua].
+
+[!DNL Oracle Eloqua] uses [Basic Authentication](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/Authentication_Basic.html) to communicate with the [!DNL Oracle Eloqua Update a Contact REST API]. Instructions to authenticate to your [!DNL Oracle Eloqua] instance are further below, in the [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, you can deliver personalized experiences to your users, based on attributes from their Adobe Experience Platform profiles. You can build segments from your offline data and send these segments to [!DNL Oracle Eloqua], to display in the users' feeds as soon as segments and profiles are updated in Adobe Experience Platform.
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Oracle Eloqua] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Refer to the Experience Platform documentation for [Segment Membership Details schema field group](/help/xdm/field-groups/profile/segmentation.md) if you need guidance on segment statuses.
+
+### [!DNL Oracle Eloqua] prerequisites {#prerequisites-destination}
+
+In order to export data from Platform to your [!DNL Oracle Eloqua] account you need to have an [!DNL Oracle Eloqua] account.
+
+#### Gather [!DNL Oracle Eloqua] credentials {#gather-credentials}
+
+Note down the items below before you authenticate to the [!DNL Oracle Eloqua] destination:
+
+| Credential | Description |
+| --- | --- |
+| `Username` | The Username of your [!DNL Oracle Eloqua] account. |
+| `Password` | The Password of your [!DNL Oracle Eloqua] account. |
+
+## Guardrails {#guardrails}
+
+>[!NOTE]
+>
+>* [!DNL Oracle Eloqua] has a maximum limit of 250 custom contact fields.
+>* [!DNL Oracle Eloqua] custom contact fields are created using the names of the segments selected during the **[!UICONTROL Destinations]** > **[!UICONTROL Activate Destination]** > **[!UICONTROL Select segments]** step.
```

Comment:
Thanks for updating this part. I just have a follow-up question. Are [!DNL Oracle Eloqua] custom contact fields created automatically in Eloqua or does the user need to create these? 

If these are created automatically, my suggestion is to add this callout to make things as clear as possible.

Reply:
They are created automatically, have reworded.